rust/src/lib/ifaces/ipsec.rs

```diff
@@ -148,6 +148,11 @@ pub struct LibreswanConfig {
     pub hostaddrfamily: Option<LibreswanAddressFamily>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub clientaddrfamily: Option<LibreswanAddressFamily>,
+    #[serde(
+        skip_serializing_if = "Option::is_none",
+        rename = "require-id-on-certificate"
+    )]
+    pub require_id_on_certificate: Option<bool>,
```

Comment:
Can you explain the use case of the setting `require_id_on_certificate` in the commit message and comments using `///` here (These comments are processed by Rust's documentation tool (rustdoc) to generate HTML documentation)  ?

Reply:
I didn't put it because the rest of the Libreswan's fields don't have this explanation. I guess it is because each field is a 1-1 match to the raw Libreswan's configs.